models/DataObject/ClassDefinition/Data/ManyToManyRelation.php

```diff
@@ -458,10 +452,7 @@ public function preGetData(mixed $container, array $params = []): mixed
 
                 $container->setObjectVar($this->getName(), $data);
                 $this->markLazyloadedFieldAsLoaded($container);
-
-                if ($container instanceof Element\DirtyIndicatorInterface) {
```

Comment:
not sure if i missed something, it's not always implementing this interface

Reply:
Container is here always a Concrete (see instanceof above). Concretes have always this interface because the AbstractElement class has it: https://github.com/pimcore/pimcore/blob/e3bae631725a9d322da80c324b219e62866770cf/models/Element/AbstractElement.php#L36